src/libs/ReportUtils.ts

```diff
@@ -4316,6 +4316,21 @@ function navigateToPrivateNotes(report: Report, session: Session) {
     Navigation.navigate(ROUTES.PRIVATE_NOTES_LIST.getRoute(report.reportID));
 }
 
+function getReportFieldTitle(report: OnyxEntry<Report>, reportField: PolicyReportField) {
+    const value = report?.reportFields?.[reportField.fieldID] ?? reportField.defaultValue;
+
+    if (reportField.type !== 'formula') {
+        return value;
+    }
+
+    return value.replaceAll(CONST.REGEX.REPORT_FIELD_TITLE, (match, property) => {
+        if (report && property in report) {
+            return report[property as keyof Report]?.toString() ?? match;
+        }
+        return match;
+    });
+}
```

Comment:
We can test this out - but I'm not sure if we can generate the formula optimistically like this. Probably some fields but we'll need to get the current result from the backend

Reply:
Are you suggesting that I should remove it?

Reply:
We can test this out - but we will have to think about a better way to handle formula titles offline if they are required. Something to bring back into a slack discussion